kobo/apps/superuser_stats/views.py

```diff
@@ -70,6 +74,27 @@ def user_report(request):
     return HttpResponse(template_ish)
 
 
+@user_passes_test(lambda u: u.is_superuser)
+def user_details_report(request):
+    base_filename = 'user-details-report_{}_{}_{}.csv'.format(
+        re.sub('[^a-zA-Z0-9]', '-', request.META['HTTP_HOST']),
+        date.today(),
+        datetime.now().microsecond,
+    )
+    filename = _base_filename_to_full_filename(
+        base_filename, request.user.username
+    )
+    generate_user_details_report.delay(filename)
+    template_ish = (
+        '<html><head><title>Hello, superuser.</title></head>'
+        '<body>Your report is being generated. Once finished, it will be '
+        'available at <a href="{0}">{0}</a>. If you receive a 404, please '
+        'refresh your browser periodically until your request succeeds.'
+        '</body></html>'
+    ).format(base_filename)
```

Comment:
I know it's kind a copy/paste from other report views but I'd rather prefer start to use `f-strings` instead of `.format()`. But it's a personal preference, no requirements to change anything here.